<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowClass</class>
 <widget class="QMainWindow" name="WindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qt DLL Injector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labDLL">
          <property name="text">
           <string>DLL Path:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="leDLL"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnBrowse">
            <property name="text">
             <string>Browse ...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labProc">
          <property name="text">
           <string>Process name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leProc"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnInject">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Inject !</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Window.qrc"/>
 </resources>
 <connections/>
</ui>
